htdocs/core/class/commondocgenerator.class.php

```diff
@@ -236,7 +236,7 @@ public function get_substitutionarray_mysoc($mysoc, $outputlangs)
 			'mycompany_idprof5'=>$mysoc->idprof5,
 			'mycompany_idprof6'=>$mysoc->idprof6,
 			'mycompany_vatnumber'=>$mysoc->tva_intra,
-			'mycompany_object'=>$mysoc->object,
+			'mycompany_object'=>$mysoc->socialobject,
```

Comment:
Rename is done on an assignement only. Property is set only, not used by a read anywhere ?

Reply:
https://github.com/Dolibarr/dolibarr/pull/23638/files/9bd3cb927dd3b3984c44c124baff58d7fb90da80#r1083589059